src/util/Util.js

```diff
@@ -145,6 +145,8 @@ define(['okta', './Logger', './Enums'], function (Okta, Logger, Enums) {
     var className = _.isFunction(controller.className) ? controller.className() : controller.className;
     var error = _.pick(err, 'name', 'message', 'statusCode', 'xhr');
     controller.trigger('afterError', { controller: className }, error);
+    // Logs to console only in dev mode
+    Logger.info('controller: ' + className + ', error: ' + error);
```

Comment:
Shouldn't this be using the `Logger.error` property instead of `info`?

Reply:
Unfortunately, `Logger.error` logs also in prod: https://github.com/okta/okta-signin-widget/blob/master/src/util/Logger.js#L18

Reply:
Want to meet in the middle and do `Logger.warn`?

Reply:
ook! 👍

Reply:
logger.warn or logger.log